ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -741,8 +742,14 @@ default List<String> showPartitions(DDLOperationContext context,
     throw new UnsupportedOperationException("Storage handler does not support show partitions command");
   }
 
-  default void validatePartSpec(org.apache.hadoop.hive.ql.metadata.Table hmsTable, Map<String, String> partitionSpec)
-      throws SemanticException {
+  /**
+   * Validates that the provided partitionSpec is a valid according to the current table partitioning.
+   * @param hmsTable {@link org.apache.hadoop.hive.ql.metadata.Table} table metadata stored in Hive Metastore
+   * @param partitionSpec Map of Strings {@link java.util.Map} partition specification
+   * @param conf Configuration
+   */
+  default void validatePartSpec(org.apache.hadoop.hive.ql.metadata.Table hmsTable, Map<String, String> partitionSpec, 
```

Comment:
you can't just change the interface, please restore and call a new API with empty `Configutation`

Reply:
Sounds good, re-implemented with the defaults.